### DISCUSSION TITLE
Reaction force 2D plane strain vs 2D axisymmetric

### DISCUSSION THREAD

**Discussion by HayThem-JDN** [Anura3D/Anura3D_OpenSource#253] [Q&A] [answered]
Dear Anura3D Developer/users,

I'm working on a 2D axisymmetric model and have a question regarding the units of the reaction force.

In a 2D plane strain model, the reaction force is typically expressed in kN/m. However, I'm unsure about the appropriate units for a 2D axisymmetric problem. Should it be kN/rad, kN/m/rad, or something else?

Your guidance on this matter would be greatly appreciated.

Thank you,
Haythem

**Comment by Francisliii** [comment, marked as answer]
It should be kN/rad. It models a wedge of one radian.

Best,
Francis
